README.md

```diff
@@ -385,6 +385,20 @@ If you want to help out, see CONTRIBUTING for a set of guidelines.
 
 This section contains general instructions for debugging failed installs or problems encountered with Monero. First ensure you are running the latest version built from the github repo.
 
+## Obtaining Stack Traces in Linux
+
+We generally use the tool `gdb` to provide stack trace functionality, althout this is currently being integrated into the easylogger++ system by @moneromooo-monero to provide stack traces across all systems
+
+To use gdb:
+
+``
+gdb /path/to/monerod `pidof monerod` 
```

Comment:
This is when getting a trace from a running process. To get a trace for a dead process: gdb /path/to/monerod /path/to/core/file  (and you need to have a core file, so you may need to run "ulimit -c unlimited" first, if it's set to 0)